src/components/prompts/TermsAndConditions.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import {
+  Button,
+  Form,
+  Modal,
+  Radio,
+} from 'semantic-ui-react';
+
+/**
+ * Displays a model that requires the user to accept the terms and conditions before using the app
+ * @extends React.Component
+ * @param {boolean} termsAccepted - boolean indicating whether terms and conditions have been accepted by the user
+ * @param {function} toggleAcceptTerms - function to set the termsAccepted in app state
+ * @param {object} snippets - object containing localization snippets
+ */
+class TermsAndConditions extends Component {
+  
+  state = { canCountOnPredictions: null };
+
+  closeModal = (accept) => {
+    if (accept) {
+      this.props.toggleAcceptTerms();
+    } else {
+      this.props.history.push('/');
+    }
+  };
+
+  handleChange = (canCountOnPredictions) => {
+    this.setState({ canCountOnPredictions });
+  };
+
+  render() {
+  
+    const {
+      termsAccepted,
+      snippets,
+    } = this.props;
+
+    return (
+      <Modal
+        id={ `WarningModal` }
+        mountNode = { document.getElementById('App') }
+        size='large'
+        open={ !termsAccepted }
+        closeOnDimmerClick={ false }
+        closeOnEscape={ false }>
+        <Modal.Header> 
+          { snippets.i_header }
+        </Modal.Header>
+        <Modal.Content scrolling>
+
+          { snippets.i_terms } 
+
+          <h4>{ snippets.i_formHeader }</h4>
+
+          <div className="radio-yes-no">
```

Comment:
This isn't actually a radio yes/no button, so maybe this class name doesn't fit. It may be worth adding another class to the styles that this needs, or maybe to make a broader class that encompasses both - `radio-inline` or something more clever?

Reply:
It doesn't make sense to have a separate class, so I used the existing.  Maybe it should just be "radio" across the board since it give the proper padding and alignment to the radios.